runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/util/common/worker/MapTaskExecutorTest.java

```diff
@@ -154,6 +154,7 @@ public void testExecuteMapTaskExecutor() throws Exception {
     inOrder.verify(o1).finish();
     inOrder.verify(o2).finish();
     inOrder.verify(o3).finish();
+    inOrder.verify(stateTracker).deactivate();
```

Comment:
would improve test to verify the activate happens before any the start methods, here and below.

Reply:
Done